Blackjack/IPlayer.cs

```diff
@@ -1,8 +1,10 @@
+using System.Collections.Generic;
+
 namespace Blackjack
 {
     public interface IPlayer
     {
-        public Hand Hand { get; }
+        public List<Card> ReceiveCard(Card card);

```

Comment:
you don't need the `public` access modifier for interfaces because they are always public by default